libs/sparkpost-lib/src/main/java/com/sparkpost/resources/ResourceTransmissions.java

```diff
@@ -22,6 +24,16 @@
 
     public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientArray trans) throws SparkPostException {
 
+        return createTransmission(conn, numRcptErrors, trans);
+    }
+
+    public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientList trans) throws SparkPostException {
```

Comment:
I think instead of two methods we could just use TransmissionBase. Do you think it is better to overload the method?